lib/minify.js

```diff
@@ -0,0 +1,168 @@
+exports.readFile = function() {
+    DefaultsError.croak("UglifyJS has no access to read from the file system");
```

Comment:
Error message is too verbose - "readFile not supported" is sufficient. Ditto for writeFile and simple_glob.